app/Console/Commands/UpdateSignupAndPostCampaignIds.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Rogue\Models\Signup;
+use Rogue\Models\Campaign;
+use Illuminate\Console\Command;
+
+class UpdateSignupAndPostCampaignIds extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:updatesignupandpostcampaignids {--campaign= : The campaign_id to update signup and posts under}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Updates signup and post campaign ids after legacy campaign migration and killing of campaign runs.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->line('rogue:updatesignupandpostcampaignids: Starting script to update campaign ids!');
+
+
+        // Grab all of the campaigns in the campaigns table.
+        $query = (new Campaign)->newQuery();
+
+        if ($this->option('campaign')) {
+            $query = $query->where('id', $this->option('campaign'));
+        }
+
```

Comment:
added this option so we can test a small batch of campaigns before running the update on all signups and posts!